src/main/resources/sql/selectAqToNwisParm.sql

```diff
@@ -0,0 +1,5 @@
+select
+    d.parm_cd,
+    d.above_datum,
+    d.below_land_surface
```

Comment:
Note to our future selves, if we need the datum at some point (NGVD89, NAVD88, Local Assumed Datum, etc...) we can grab it from this call to the datum column.

Reply:
Hey future selves. We need the datum for all rows above a datum.